packages/server/src/insights/insights.service.spec.ts

```diff
@@ -98,6 +98,54 @@ describe('InsightsService', () => {
       });
       expect(res.output).toEqual('5 min');
     });
+
+    it('averageHelpingTime', async () => {
+      const question = await QuestionFactory.create({
+        helpedAt: new Date(Date.now() - 30 * 60 * 1000),
+        closedAt: new Date(Date.now() - 25 * 60 * 1000),
+      });
+
```

Comment:
Could you add one more question here with a different helping time so we can see that the average function is working as expected?

Reply:
Will do!